packages/scandipwa/src/component/FieldSelect/FieldSelect.style.scss

```diff
@@ -108,14 +109,15 @@ $select-arrow-width: 14px !default;
 
     &-Option {
         cursor: pointer;
-        padding: 0;
+        padding: 0px;
         margin: 0;
         z-index: 1;
         transition: 100ms max-height ease-in;
         will-change: max-height;
-        max-height: $select-option-height;
-        line-height: 36px;
-        padding-inline-start: 18px;
+        min-height: $select-option-height;
+        line-height: 16px;
+        padding-block: 10px 10px;
```

Comment:
Is it necessary to repeat 10px in this case? I guess writing 1 time 10px will be same result.